.github/workflows/integration-test.yml

```diff
@@ -4,16 +4,18 @@
 name: Run Integration Tests
 env:
   PRIVATE_KEY: ${{ secrets.AWS_PRIVATE_KEY  }}
-  TERRAFORM_AWS_ASSUME_ROLE: ${{ secrets.TERRAFORM_AWS_ASSUME_ROLE }}
+  TERRAFORM_AWS_ASSUME_ROLE: ${{ vars.TERRAFORM_AWS_ASSUME_ROLE }}
```

Comment:
We need to use vars since secrets can't be passed into reusable yaml without inherit, but that leaves the problems that we need to inherit 2 diff values for the same secret